src/many-identity-webauthn/src/verifier.rs

```diff
@@ -0,0 +1,473 @@
+use coset::cbor::value::Value;
```

Comment:
Did you change anything at all in this file? It looks like a copy of my code, and I won't review it if no change has been made.

Reply:
No, just moved from `lib.rs`.

Reply:
Actually now I just did with the Challenge type.